internal/providers/terraform/azure/testdata/storage_account_test/storage_account_test.golden

```diff
@@ -89,56 +89,153 @@
- ├─ List operations                               Monthly cost depends on usage: $0.0715 per 10k operations  
- ├─ Read operations                               Monthly cost depends on usage: $0.013 per 10k operations   
- ├─ All other operations                          Monthly cost depends on usage: $0.00572 per 10k operations 
+ ├─ Write operations                              Monthly cost depends on usage: $0.13 per 10K operations    
+ ├─ List operations                               Monthly cost depends on usage: $0.0715 per 10K operations  
+ ├─ Read operations                               Monthly cost depends on usage: $0.013 per 10K operations   
+ ├─ All other operations                          Monthly cost depends on usage: $0.00572 per 10K operations 
  ├─ Data retrieval                                Monthly cost depends on usage: $0.01 per GB                
  └─ Early deletion                                Monthly cost depends on usage: $0.0228 per GB              
                                                                                                              
- OVERALL TOTAL                                                                                 $1,778,552.71 
+ azurerm_storage_account.v2_cool_gzrs                                                                        
+ ├─ Read operations                                               10  10K operations                   $0.10 
+ ├─ All other operations                                         100  10K operations                   $0.44 
+ └─ Data retrieval                                             1,000  GB                              $10.00 
+                                                                                                             
+ azurerm_storage_account.v2_cool_lrs                                                                         
+ ├─ Capacity                                               1,000,000  GB                          $15,200.00 
+ ├─ Write operations                                             100  10K operations                  $10.00 
+ ├─ List and create container operations                         100  10K operations                   $5.50 
+ ├─ Read operations                                               10  10K operations                   $0.10 
+ ├─ All other operations                                         100  10K operations                   $0.44 
+ ├─ Data retrieval                                             1,000  GB                              $10.00 
+ ├─ Blob index                                                    10  10K tags                         $0.39 
+ └─ Early deletion                                             1,000  GB                              $15.20 
+                                                                                                             
+ azurerm_storage_account.v2_cool_lrs_nfsv3                                                                   
```

Comment:
Is it correct they don't have support fo this in the Azure pricing calculator?

Reply:
Seems like so, I couldn't find anywhere how to "enable" it in the calculator. I used the pricing page to compare the results.